src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bladeType;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String bladeType, String name, String color) {
+        this.bladeType = bladeType;
+        super.setName(name);
+        super.setName(color);
```

Comment:
another option is to create the constructor in the Machine class and call it here